app.rb

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env ruby
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def print_question
+    puts 'Welcome to School library App! 🎉'
+    puts "Please choose an option by enterin a number:
+      1- List all books.
+      2- List all people.
+      3- Create a person.
+      4- Create a book.
+      5- Create a rental.
+      6- List all rentals for a given person id.
+      7- Exit."
+  end
+
+  def initialize
+    @books = []
+    @person = []
+    @rentals = []
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [input the number]'
+    num = gets.chomp
+    num = num.to_i
+
+    print 'age:'
+    age = gets.chomp
+    age = age.to_i
+
+    print 'Name:'
+    name = gets.chomp
+
+    case num
+    when 1
+      print 'Has parent permission? [y/n]:'
+      parent_permission = gets.chomp
+      case parent_permission
+      when 'y' then parent_permission = true
+      when 'n' then parent_permission = false
+      end
+      @person.push(Student.new(age, name, parent_permission: parent_permission))
+    when 2
+      print 'Specialization:'
+      specialization = gets.chomp
+      @person.push(Teacher.new(age, specialization, name))
+    else
+      puts 'Invalid number, please enter number again!'
+    end
+    puts 'Person created successfully 🎉🎉'
+  end
+
+  def list_people
+    @person.each do |per|
+      puts "[#{per.class}] Name: #{per.name}, ID: #{per.id}, Age: #{per.age}"
+    end
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def create_book
+    print 'Title:'
+    title = gets.chomp
+
+    print 'author:'
+    author = gets.chomp
+
+    @books.push(Book.new(title, author))
+
+    puts 'Book created successfully 🎉🎉 '
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number '
+    list_books
+    book_num = gets.chomp
+    book_num = book_num.to_i
+
+    puts 'Select a person from the following list by number (not id)'
+    list_people
+    person_num = gets.chomp
+    person_num = person_num.to_i
```

Comment:
- [ ] You are asking for selecting a book and a person by a number, but there are no numbers next to the books list and people list. I would recommend adding numbers so the user is able to choose a book or person by number. I had to guess some numbers but the console shows me an error. Please try to work on that.

<img width="550" alt="image" src="https://user-images.githubusercontent.com/77190203/162500912-f1be0587-2a06-4fbd-ba8c-5ece51107509.png">